pkg/packagekit/updates.jsx

```diff
@@ -480,15 +480,15 @@ class ApplyUpdates extends React.Component {
         }
 
         return (
-            <div>
+            <React.Fragment>
                 <div className="progress-main-view">
                     <div className="progress-description">
                         <div className="spinner spinner-xs spinner-inline" />
                         {actionHTML}
                     </div>
                     <div className="progress progress-label-top-right">
                         <div className="progress-bar" role="progressbar" style={ {width: this.state.percentage + "%"} }>
-                            { this.state.timeRemaining !== null ? <span>{moment.duration(this.state.timeRemaining * 1000).humanize()}</span> : null }
+                            { this.state.timeRemaining !== null ? <React.Fragment>{moment.duration(this.state.timeRemaining * 1000).humanize()}</React.Fragment> : null }
```

Comment:
I'm not sure about having a fragment in a fragment. Is this correct?

Reply:
According to https://www.patternfly.org/pattern-library/widgets/#progress-bars it needs to be a `<span>`. Maybe it works without, but I'd rather stick with the official documentation.

Reply:
Reverted this to `<span>`